libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -633,6 +635,45 @@ void AP_DDS_Client::on_request(uxrSession* uxr_session, uxrObjectId object_id, u
         GCS_SEND_TEXT(MAV_SEVERITY_INFO, "%s Request for Mode Switch : %s", msg_prefix, mode_switch_response.status ? "SUCCESS" : "FAIL");
         break;
     }
+    case services[to_underlying(ServiceIndex::GEOFENCE_REQUEST)].rep_id: {
+        ardupilot_msgs_srv_GeoFence_Request geofence_request;
+        ardupilot_msgs_srv_GeoFence_Response geofence_data;
+
+        const bool deserialize_success = ardupilot_msgs_srv_GeoFence_Request_deserialize_topic(ub, &geofence_request);
+        if (deserialize_success == false) {
+            break;
+        }
+        
+        if(geofence_request.req)
+        {
+            geofence_data.radius = AP::fence()->get_radius();
```

Comment:
While ardupilot supports both fence types of `AC_FENCE_TYPE_CIRCLE` and `AC_FENCE_TYPE_POLYGON`, this message population seems to assume it's a circle. 

Also, what is the plan for the `PolyFence` support in ArduPilot, which has inclusion/exclusions.